schema/subnetInfoSchema.json

```diff
@@ -14,6 +14,45 @@
     },
     "isTestnet": {
       "type": "boolean"
+    },
+    "description": {
+      "type": "string"
+    },
+    "officialSite": {
+      "type": "string",
+      "format": "uri"
+    },
+    "resourceLinks": {
```

Comment:
Not sure if this is worth the extra effort but if we use this sub-schema multiple times we might want to extract this and reuse it when applicable

Reply:
Yeah agreed that reusing the schemas would be better, but I'm not super familiar with ajv schemas and their docs are a bit confusing - do you have any examples of how this would work?

If it's not an easy lift - is it alright with you if we punt this?  I have a lot on my plate right now and perfecting our schemas feels like a pretty low priority thing for me, especially since it could be added later without any breaking change.  I'd be happy to circle back later to fix this though if needed.